website/docs/docs/building-a-dbt-project/projects.md

```diff
@@ -38,19 +38,15 @@ To create a new dbt project when developing in dbt Cloud:
 
 1. Create a dbt Cloud account [here](https://cloud.getdbt.com/signup/). If your organization already has a dbt Cloud account, ask an admin to add you as a Developer.
 2. If you created a new account, a new project should automatically be created. If you were added to an existing account:
-    * Click the hamburger menu, then `Account Settings`, then `Projects`.
-    * Name your project, and click `Save`. There's no need to fill in the other details.
-    * Click the hamburger menu, and then `Home`.
-    * Switch the project in the header bar to your new "dbt Tutorial" project.
+    * Click the gear icon in the top right of the screen, then `Projects` underneath your Account Settings.
```

Comment:
Let's use bold so it doesn't get confused with code. (I know this is what we had there, but I can't help myself). Also just shortening the text.
```suggestion
    * Click the gear icon to the top right, then **Projects** under Account Settings.
```